src/components/QRCode/QRCode.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import qrImage from 'qr-image';
+import { connectTheme } from '../../utils';
+
+/*
+* Taken from:
+* https://github.com/jprichardson/react-qr
+* */
+
+class QRCode extends Component {
+  constructor(props) {
+    super(props);
+  }
+  static get propTypes() {
+    return {
+      text: PropTypes.string,
+      containerStyle: PropTypes.object
+    };
+  }
+  render() {
+    const { text, containerStyle } = this.props;
+    if (text) {
+      const pngBuffer = qrImage.imageSync(text, { type: 'png', margin: 1 });
+      const dataURI = `data:image/png;base64,${pngBuffer.toString('base64')}`;
+      return (
+        <div style={containerStyle}>
```

Comment:
Does it make sense to have the style be something that can come from the theme? That might be good for any default styles. You can look the [Text component](https://github.com/bpanel-org/bpanel-ui/blob/132103eb3b79c27457545284e8688fc578cf6172/src/components/Text/Text.js) for how this works in a way that also supports passing down styles if you want. If qr code component also supports custom coloring this could be a good opportunity to use the theme to custom color the qr code based on the theme (though with `connectTheme` that could be done either way by passing through with style I suppose).

Reply:
Updated to follow that convention